acm/src/main/java/org/literacybridge/acm/store/AudioItemIndex.java

```diff
@@ -380,6 +361,24 @@ private Playlist getPlaylist(final String uuid, final Playlist playlist) throws
         }
     }
 
+    private void loadPlaylistFromPostingList(LeafReader leafReader, String playlistUuid, Playlist.Builder playlistBuilder) throws IOException {
+        PostingsEnum tp = leafReader.postings(new Term(PLAYLISTS_FIELD, playlistUuid), PostingsEnum.PAYLOADS);
+        if (tp != null) {
+            while (tp.nextDoc() != DocIdSetIterator.NO_MORE_DOCS) {
+                // important: Lucene applies deletes of documents to posting lists lazily when it
+                // performs a segment merge, so it is necessary here to check if this audioItem is deleted
+                if (leafReader.getLiveDocs() == null || leafReader.getLiveDocs().get(tp.docID())) {
+                    // TODO: we could also use the termPosition instead of a payload to store the playlist position
```

Comment:
Should we use the termPosition? Will we ever do so? If not, the comment as it stands doesn't really add much value. It could be useful to document why we don't use a termPosition, but perhaps that should be at a higher level.

And if we should use a termPosition, please file an issue. Nobody ever, ever comes back and cleans up TODOs in code. :(